pkg/errs/errno.go

```diff
@@ -271,6 +271,11 @@ var (
 	ErrSemverNewVersion = errors.Normalize("new version error", errors.RFCCodeText("PD:semver:ErrSemverNewVersion"))
 )
 
+// config
+var (
```

Comment:
Why the error is `PD:filepath:ErrFilePathAbs`?

Reply:
updated. PTAL